### PR TITLE
test case optimized code

### PR DESCRIPTION
**Pull request by jiangxinlingdu**
find test case one if never be executed，optimized code

### CODE REVIEW

File: lib/storage/storage_test.go [hunk 1]
```diff
@@ -961,9 +961,6 @@ func testStorageRegisterMetricNames(s *Storage) error {
 		return fmt.Errorf("unexpected error in TagFilters.Add: %w", err)
 	}
 	metricNames, err := s.SearchMetricNames(nil, []*TagFilters{tfs}, tr, metricsPerAdd*addsCount*100+100, noDeadline)
-	if err != nil {
-		return fmt.Errorf("error in SearchMetricNames: %w", err)
-	}
```
Comment (dmitryk-dk) [inline lib/storage/storage_test.go:966]: I think it is better to remove the second unused error. Because this one told you about real problem

Comment (jiangxinlingdu) [reply lib/storage/storage_test.go]: ok

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by hagen1778 — APPROVED** [review]
Thank you!

**Pull request merged**